src/styles/blocks/rates.scss

```diff
@@ -0,0 +1,30 @@
+.rates {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 16px;
+
+  &__stars {
+    display: flex;
+    flex-direction: row;
+  
+    &__star {
+      height: 16px;
+      margin-right: 4px;
+      width: 16px;
+      background-image: url(./../../images/star.svg);
+      background-repeat: no-repeat;
+      background-position: center;
+    }
+  
+    &:nth-child(-n + 4) {
+      background-image: url(./../../images/star-active.svg);
+    }
+  }
```

Comment:
You had to create a function to make a modifier for all counts of stars. For instance, if you change the counts of stars in your HTML file, your code will be broken